File: jsReview/lab1/fizzBuzz.js
let a = parseInt(prompt('Enter a number')) 
let b = parseInt(prompt('Enter a another number')) 
​
for (let i = 1; i <= 100; i++) {
    let str = ''
    if (i % 3 === 0) str += 'Fizz' 
    if (i % 5 === 0) str += 'Buzz' 
​
    console.log(str || i)
}

File: ruby/topics/fizzbuzz.rb
num = gets.to_i

for number in 1..num
    if number % 3 == 0 && number % 5 == 0 
        puts "fizzbuzz"
    elsif number % 3 == 0
        puts "fizz"
    elsif number % 5 == 0
        puts "buzz"
    else 
        puts number
    end
end